src/partials/auth-security.md

```diff
@@ -24,4 +24,12 @@ If an user is imported into Appwrite with hash differnt than Argon2, the passwor
 Encourage passwords that are hard to guess by disallowing users to pick passwords that contain personal data.
 Personal data includes the user's name, email, and phone number.
 
-Disallowing personal data can be enabled in the Auth service's **Security** tab on the Appwrite Console.
\ No newline at end of file
+Disallowing personal data can be enabled in the Auth service's **Security** tab on the Appwrite Console.
+
+# Session alerts {% #session-alerts %}
+
+Enable email alerts for your users so that whenever another session is created for their account, they will be alerted to the new session.
+
+Users must have a valid email address to receive email notifications. But you don't receive notifications when logging in using [Magic URL](/docs/products/auth/magic-url), [Email OTP](/docs/products/auth/email-otp), or [OAuth2](/docs/products/auth/oauth2). 
```

Comment:
> Users must have a valid email address to receive email notifications. 

Let's actually leave this out because it's kind of self-explanatory

> But you don't receive notifications

Saying "but" like this sounds a little awkward/out of place. Maybe it would be helpful to expand this by explaining a bit. The goal of session alerts is to notify users if someone unauthorized logged into their account. We probably decided to not send session alerts for magic URL, email OTP, and OAuth2 because those are more likely to be legitimate auths since they must have access to these other systems rather than just a password.